pkg/queue/consumers/kafka/consumer.go

```diff
@@ -0,0 +1,96 @@
+package kafka
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+)
+
+const (
+	POLL_INTERVAL = 1
+)
+
+type Consumer struct {
+	consumer           *kafka.Consumer
+	UncommitedMessages []string
+}
+
+type ConsumerConfig struct {
```

Comment:
The configs can be specific - but we can use interface{} to pass them. Otherwise we have to forcefully fit the configs - with some fields having different names and some not available needed for a specific config.